client/my-sites/stats/stats-list/stats-list-posts.jsx

```diff
@@ -0,0 +1,133 @@
+import {
+	// eslint-disable-next-line import/named
+	HorizontalBarList,
```

Comment:
I'm not sure why isn't this recognised. Lint doesn't see it but the exports are there...

Reply:
It works fine for me without the `eslint` override comments on my local machine 🤔

Reply:
I will double-check that. Maybe it's just my linter or IDE setup.

Reply:
Downgrading `typescript` from 4.7.4 to 4.7.2 resolved the issue but since 4.7.4 is in `package.json` it's not a great solution... What version are you using?

Reply:
Strange, I'm using 4.7.4 as well but not seeing any errors.